package.json

```diff
@@ -31,7 +31,8 @@
     "next": "11.0.1",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-hook-form": "7.9.0"
+    "react-hook-form": "7.9.0",
+    "yarn": "^1.22.10"
```

Comment:
Why would you install yarn?

Reply:
For some reason when I run it on my laptop it wasnt working unless i install it there were defect in that laptop it should run after removing those